### ISSUE TITLE
TypeError: '>' not supported between instances of 'int' and 'NoneType' when num_qubits = 1

### ISSUE THREAD

**Issue by MarcelRobitaille** [Qiskit/qiskit-metapackage#1033]
## Informations

- **Qiskit version**: 0.20.0
- **Python version**: 3.8.5
- **Operating system**: `Linux Zeus 5.4.61-1-lts #1 SMP Wed, 26 Aug 2020 18:35:48 +0000 x86_64 GNU/Linux`

### What is the current behavior?
When initializing a `QuantumCircuit` with one qubit (`QuantumCircuit(1)`), `plot_block_multivector` raises the error:
```
TypeError: '>' not supported between instances of 'int' and 'NoneType'  
```

<details> 
  <summary>Full stack trace</summary>
TypeError                                 Traceback (most recent call last)
<ipython-input-128-66f6c58a3f9e> in <module>
      6 backend = Aer.get_backend('statevector_simulator')
      7 out = execute(qc, backend).result().get_statevector()
----> 8 plot_bloch_multivector(out).show()

/tmp/venv/lib/python3.8/site-packages/qiskit/util.py in wrapper(*args, **kwargs)
    108             if kwargs:
    109                 _rename_kwargs(func.__name__, kwargs, kwarg_map)
--> 110             return func(*args, **kwargs)
    111         return wrapper
    112     return decorator

/tmp/venv/lib/python3.8/site-packages/qiskit/visualization/state_visualization.py in plot_bloch_multivector(state, title, figsize, rho)
    269                           'matplotlib".')
    270     # Data
--> 271     bloch_data = _bloch_multivector_data(state)
    272     num = len(bloch_data)
    273     width, height = plt.figaspect(1/num)

/tmp/venv/lib/python3.8/site-packages/qiskit/visualization/utils.py in _bloch_multivector_data(state)
    310     bloch_data = []
    311     for i in range(num):
--> 312         paulis = PauliTable(np.zeros((3, 2 * (num - 1)), dtype=np.bool)).insert(
    313             i, pauli_singles, qubit=True)
    314         bloch_state = [np.real(np.trace(np.dot(mat, rho.data))) for mat in paulis.matrix_iter()]

/tmp/venv/lib/python3.8/site-packages/qiskit/quantum_info/operators/symplectic/pauli_table.py in insert(self, ind, value, qubit)
    323
    324         # Column insertion
--> 325         if ind > self.num_qubits:
    326             raise QiskitError("Index {} is greater than number of qubits"
    327                               " in the PauliTable ({})".format(ind, self.num_qubits))

TypeError: '>' not supported between instances of 'int' and 'NoneType'

</details>

### Steps to reproduce the problem
```
from qiskit import QuantumCircuit, execute, Aer
from qiskit.visualization import plot_bloch_multivector

qc = QuantumCircuit(1)

backend = Aer.get_backend('statevector_simulator')
out = execute(qc, backend).result().get_statevector()
plot_bloch_multivector(out).show()
```

### What is the expected behavior?
It should plot the bloch vector without error, as it does when `num_qubits=2`.

### Suggested solutions
Looks like `qiskit/quantum_info/operators/symplectic/pauli_table.py` doesn't get set with `num_qubits` when this value is 1. I have not gone through all the code between `QuantumCircuit.__init__` and `PauliTable.insert` to debug why this is. Quick and dirty hack could just be to change the if statement on line 325 to `if self.num_qubits and ind > self.num_qubits`.

**Comment by kdk** [comment]
Thanks for reporting @Iambecomeroot . A fix for this issue has has been merged into terra's development branch, and will be included in the next qiskit minor release. For more info, see https://github.com/Qiskit/qiskit-terra/pull/4948 .

**Comment by theheavygluon** [comment]
This has been happening to me ever since I upgraded to 0.20. When will this get fixed?

**Comment by kdk** [comment]
@theheavygluon The fix for this was included in yesterday's 0.20.1 release.
